docs/docs/capturing/index.md

```diff
@@ -116,6 +116,103 @@ Tracee can capture the following types of artifacts:
+     pcap/processes/host
+     pcap/processes/host/node_186708_1573567360495399.pcap
+     pcap/processes/host/node_1196826_1662656211119567.pcap
+     pcap/processes/host/zerotier-one_7882_137007714376.pcap
+     pcap/processes/host/sshd_1196773_1662654999660718.pcap
+     pcap/processes/fd95a035ce5
+     pcap/processes/fd95a035ce5/ping_1261163_1663769383806467.pcap
+     pcap/containers
+     pcap/containers/host.pcap
+     pcap/containers/b86533d11f3.pcap
+     pcap/containers/fd95a035ce5.pcap
+     ```
+
+     you can see the packets by executing tcpdump on any pcap file:
+
+     ```text
+     $ tcpdump -r pcap/containers/b86533d11f3.pcap
+     reading from file pcap/containers/b86533d11f3.pcap, link-type NULL (BSD loopback), snapshot length 65535
+     02:52:00.524035 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 476, length 64
+     02:52:00.533145 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 476, length 64
+     02:52:01.525455 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 477, length 64
+     02:52:01.535414 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 477, length 64
+     02:52:02.526715 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 478, length 64
+     02:52:02.536444 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 478, length 64
+     02:52:03.528739 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 479, length 64
+     02:52:03.538622 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 479, length 64
+
+     $ tcpdump -r pcap/commands/b86533d11f3/ping.pcap
+     reading from file pcap/commands/b86533d11f3/ping.pcap, link-type NULL (BSD loopback), snapshot length 65535
+     02:52:00.524035 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 476, length 64
+     02:52:00.533145 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 476, length 64
+     02:52:01.525455 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 477, length 64
+     02:52:01.535414 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 477, length 64
+     02:52:02.526715 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 478, length 64
+     02:52:02.536444 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 478, length 64
+     02:52:03.528739 IP 172.17.0.3 > dns.google: ICMP echo request, id 5, seq 479, length 64
+     02:52:03.538622 IP dns.google > 172.17.0.3: ICMP echo reply, id 5, seq 479, length 64
+     ```
+
+    !!! Note
+        Note that the same packets were written to 2 different pcap files: the
+        pcap file describing the container `b86533d11f3` (because it was
+        executing a single process: ping) and the pcap file describing ANY ping
+        command executed in that container (commands/b86533d11f3/ping.pcap).
+
+1. **Network PCAP Files (deprecated)**
+
      Anytime a **packet** goes through the **network interface**, the **packet
      is captured into the pcap file**. Only packets that are generated by traced
```

Comment:
This is not true. All packets are being captured, no matter the traced events. Will need some coordination with multiple scope changes IF we want to limit the packets being written to pcap files using the filters.

Need to clarify that here.